hal/armv7m/imxrt/10xx/_startup.S

```diff
@@ -97,11 +97,11 @@ _fcfb:
 
 .word 0x00000100 /* pageSize */
 .word 0x00001000 /* sectorSize */
-.word 0x00000000 /* ipCmdSerialClkFreq */
+.word 0x00000001 /* ipCmdSerialClkFreq */
 .word 0x00000000
 
 /* reserved */
-.word 0x00000000, 0x00000000, 0x00000000, 0x00000000
+.word 0x00010000, 0x00000000, 0x00000000, 0x00000000
```

Comment:
Why change reserved value?

Reply:
This fcfb is written based on the user manual. There, this section is assigned as reserved.
However, when I looked into the ROM API, this reserved field contains block_size and is part of `flexspi_norConfig_t` structure. 
When the fcfb configurations are copied by flash driver, this reserved field is cast into `flexspi_norConfig_t`.